apps/frontend/src/libs/modules/store/libs/middlewares/handle-redirect.middleware.ts

```diff
@@ -0,0 +1,22 @@
+import { isAction, type Middleware } from "@reduxjs/toolkit";
+
+import { AppRoute } from "~/libs/enums/enums.js";
+import { store } from "~/libs/modules/store/store.js";
+import { actions as appActions } from "~/modules/app/app.js";
+import { actions as authActions } from "~/modules/auth/auth.js";
+
+const handleRedirectMiddleware: Middleware = () => {
+	return (next) => (action) => {
+		if (
+			isAction(action) &&
+			(action.type === authActions.resetPassword.fulfilled.type ||
+				action.type === authActions.checkResetPasswordExp.rejected.type)
```

Comment:
initial intent was to make a switch statement, but linter requires 3 cases for it, and it doesn't like cases with the same logic inside of them